src/main/java/frc/robot/subsystems/LimelightPortal.java

```diff
@@ -17,9 +17,10 @@
    * Creates a new LimelightPortal.
    */
   public LimelightPortal() {
-     shooterLL = NetworkTableInstance.getDefault().getTable("limelight-shooter"); 
-     shooterLL.getEntry("stream").setNumber(0);
+    shooterLL = NetworkTableInstance.getDefault().getTable("limelight-shooter"); 
```

Comment:
attempted to set the stream mode, didn't work but keep the code nonetheless, review what we do need after stream mode problem is fixed